docs/custom_regions.qmd

```diff
@@ -0,0 +1,142 @@
+# How to Define Custom Regions for DPS Jobs
+This doc explains how to define a new region, how to customize which settings it uses, how to run a region using the `manual v3` workflow, and how to create a new scheduled job for your region.
+
+
+## How DPS Jobs Pick Up Custom Settings
+
+There are several settings, such as EPSG code, that need to be changed depending on the region for which we are running FEDS. The default settings are those stored in `FireConsts.py:Settings`. 
+Most custom regions will require some changes to these default settings.
+
+The settings used to run a given region `regnm` on DPS are defined in a `.env` file stored in the `Settings.PREPROCESSED_DIR/${regnm}` folder. If there is no such file here, the default settings are used. 
+
+::: callout-note
+`Settings.PREPROCESSED_DIR` renders out to be `s3://maap-ops-workspace/shared/gsfc_landslides/FEDSpreprocessed` in our current production setup.
```

Comment:
this is nitpicky, but "renders" feels like an odd word choice. When I hear "render" I think of something that is loading on the fly. Is it possible to say "points to" or "is defined as"?